test/unit/plugins/multientity-plugin_test.js

```diff
@@ -104,7 +104,7 @@ var iotAgentLib = require('../../../lib/fiware-iotagent-lib'),
 
```

Comment:
This file seems not been in the original PR... maybe it has been leaked in the last commit?

Reply:
Sorry!!! You are right! Fixed in c7ac799e0cc31ee2326f2f81bf29fc541f94eb26